src/server/modules/auth/email/email.router.ts

```diff
@@ -21,7 +22,9 @@ export const emailSessionRouter = router({
     .mutation(async ({ ctx, input: { email } }) => {
       // TODO: instead of storing expires, store issuedAt to calculate when the next otp can be re-issued
       // TODO: rate limit this endpoint also
-      const expires = new Date(Date.now() + env.OTP_EXPIRY * 1000)
+      const expires = new Date(
```

Comment:
instead of changing the time here, should we instead use `formatInTimeZone` below in the `sendMail` body?

Reply:
Do you also want to attempt the TODO? Requires a schema change, hehe!